src/components/Options.tsx

```diff
@@ -138,14 +138,14 @@ export default class Options<OptionType> extends React.Component<
         ]}
       >
         {props.type === "checkbox" ? (
-          <Controls<OptionType>
+          <Controls
```

Comment:
`OptionType` not being used here?

Reply:
explicit type params only needed when props/args are insufficient to determine type or cause ambiguity

example: in `<Select />`, type for `options` is union `T | T[]` where `T` is not restricted. for some array type `Data[]`, how should the above union be resolved?

- `T[]` where `T = Data`
- `T` where `T = Data[]`

https://github.com/anarock/pebble-native/blob/1369d8c1c6b7f49ea812632996c6a425421fd123/src/components/Select.tsx#L221-L231